lib/fathead/mdn_css/parse.pl

```diff
@@ -296,7 +296,7 @@ sub create_redirects {
 
     # Capture content inside parentheses
     # E.g. "::before (:before)"
-    if ( $title =~ m/(::.+) \((:.+)\)/ ) {
+    if ( $title =~ m/(.+) \((:.+|-webkit.+)\)/ ) {
```

Comment:
@rasikapohankar I think we're going to need to handle the `-moz` and `-webkit` names with `()` in them slightly better. We're generating some odd redirects:

E.g. `-moz-tree-rowhover     R       :-moz-tree-row(hover)`

This should probably be: `-moz-tree-row hover     R       :-moz-tree-row(hover)`

Similarly, we're creating duplicates now as well:

- `-moz-placeholder       R       :-moz-placeholder`
- `-moz-placeholder       R       ::-moz-placeholder`

In those cases, we should actually generate a disambiguation.